### DISCUSSION TITLE
Can we get clean build for Spark OS ?

### DISCUSSION THREAD

**Discussion by Burhanverse** [ROM-builders/temporary#21752] [Q&A]
I am getting this error,
Failing repos:
hardware/qcom-caf/sm8150/display
vendor/qcom/opensource/commonsys-intf/display

I guess this is a cache issue as i am trying to build for Lancelot which is a mtk device and i can't relate this error with anything.

**Comment by Apon77** [comment]
build link?

**Reply by Burhanverse** [reply]
Here : https://cirrus-ci.com/build/6313640219901952
